Source/components/Page.js

```diff
@@ -0,0 +1,147 @@
+import { DefinedPropertiesKey } from '../WrappedObject'
+import { Group } from './Group'
+import { Selection } from '../Selection'
+import { Rectangle } from '../Rectangle'
+import { Types } from '../enums'
+import { Factory } from '../Factory'
+import { DEFAULT_EXPORT_OPTIONS } from './Layer'
+import { wrapNativeObject, wrapObject } from '../wrapNativeObject'
+
+/**
+ * Represents a Page in a Sketch document.
+ */
+export class Page extends Group {
+  /**
+   * Make a new page object.
+   *
+   * @param {MSPage} page The underlying model object from Sketch.
+   */
+  constructor(page = {}) {
+    if (!page.sketchObject) {
+      // eslint-disable-next-line no-param-reassign
+      page.sketchObject = Factory.createNative(Page)
+        .alloc()
+        .initWithFrame(new Rectangle(0, 0, 100, 100).asCGRect())
+    }
+
+    super(page)
+  }
+
+  /**
+   * The layers that the user has selected.
+   *
+   * @return {Selection} A selection object representing the layers that the user has selected.
+   */
+  get selectedLayers() {
+    return new Selection(this)
+  }
+
+  /**
+   * Export this layer (and the ones below it), using the options supplied.
+   *
+   * @param {dictionary} options Options indicating which layers to export, which sizes and formats to use, etc.
```

Comment:
Yeah I was actually thinking about having `export` as a top-level method instead of a method on layers and pages, like `sketch.export(layer, options)` and `sketch.export([layers])`.

What do you think?